### PR TITLE
fix(cordova/apple/ios): reset signing

### PR DESCRIPTION
**Pull request by fortuna**
This makes the command-line build succeed, as well as the manual signing.

### CODE REVIEW

**Comment by fortuna** [comment]
@daniellacosse I think master/ci is broken

**Comment by fortuna** [comment]
This PR actually works, and it's what I used to release to Test Flight.

File: src/cordova/apple/xcode/ios/Outline.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -594,7 +590,8 @@
 				CLANG_ENABLE_MODULES = YES;
 				CLANG_ENABLE_OBJC_ARC = YES;
 				CODE_SIGN_ENTITLEMENTS = Outline/Outline.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
+				CODE_SIGN_IDENTITY = "Apple Development";
```
Comment (daniellacosse) [inline src/cordova/apple/xcode/ios/Outline.xcodeproj/project.pbxproj:593]: You're still setting the CSI to "Apple Development", just like my PR

**Review by daniellacosse — APPROVED** [review]
Windows thing is broken on master, so you can merge

**Pull request merged**